apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md

```diff
@@ -0,0 +1,107 @@
+
+
+## Enhancing High-Level Scientific Computing Support in CuPy
+**PI: [Kenichi Maehashi](https://github.com/kmaehashi) (Preferred Networks)**
+
+Led by [Preferred Networks](https://www.preferred.jp/en), Quansight Labs is proud to collaborate on this grant, which aims to provide a series of GPU accelerated routines for signal processing and interpolation in CuPy. The goal is to match SciPy’s `signal` and `interpolate` submodules,  as a foundation for the research community. 
+
+This project has two main goals
+1. Develop interpolation and signal processing modules
+2. Maintain CuPy to continue supporting the latest platforms
+
+### Develop interpolation and signal processing modules
+The CuPy project aims to reach full [coverage of the NumPy and SciPy APIs](https://docs.cupy.dev/en/latest/reference/comparison.html). The developers will implement, profile, document, and add unit tests for each API in the targeted modules: `cupyx.scipy.interpolate` and `cupyx.scipy.signal`. This work includes:
+
+
+* Careful analysis of the underlying algorithms will be carried out, and specialized GPU kernels (CUDA C/C++) will be written from scratch when needed.
+* GPU & CPU elapsed implementation time profiling for a variety of GPU devices representing common research environments 
+* Unit tests added to cover all possible input combinations and edge cases, e.g. overflows, exceptions, and warnings
+
+### Maintain CuPy to continue supporting the latest platforms
+As with other GPU applications, CuPy heavily relies on NVIDIA CUDA and AMD ROCm libraries. Currently, CuPy supports with CUDA versions (10.2, 11.0-11.6) and four ROCm versions (4.0, 4.2, 4.3, 5.0). GPU vendors generally release updates of these toolkits on a quarterly basis, and providing immediate support for them is essential to allow users to enjoy the latest features, performance enhancements, security, and bug fixes. This work includes:
+
+* Implementing Python wrappers for new APIs added to the library to make them available to CuPy
+* Supporting  new features from a new CUDA or ROCm release, when applicable 
+* Setting up a continuous integration system to provide extensive coverage of the supported software combinations
+* Building a delivery pipeline to provide binary packages for the supported versions of the library
+
+
+# Want to join?
```

Comment:
@gabalafou the spacing here seems a bit short to me. Is this a known issue with a header after a list perhaps?
<img width="1005" alt="Screenshot of the heading Want to Join?" src="https://user-images.githubusercontent.com/6564007/201186856-a8182774-3599-4cb7-bc77-c53e0eeeea78.png">

Reply:
For now we can insert a `<br />` to put more space between the list and the heading.

```suggestion
<br />

# Want to join?
```

Reply:
I'm pretty sure this issue isn't relevant to the Consulting site, at least up to the present. The Storyblok content editor has given us a fair measure of control over vertical spacing like this. (It's sometimes a bit awkward to make it produce the vertical spacing we want, but it's usually possible.)

Once #396 is completed, though, then this _will_ be relevant to the Consulting blog also. 

---

I'm a bit hesitant to declare this a uniform styling problem with bulleted lists.  How is the spacing for a bulleted list followed by a normal paragraph? If it looks ok, then this is likely just an issue for when a bulleted list ends a section. If so, I think we'd want any style fix to be narrowly scoped, or else the spacing might be too *big* in some cases---and that would be harder to patch in Markdown than too little space.